test/suites/incus_user.sh

```diff
@@ -0,0 +1,17 @@
+test_incus_user() {
+  ensure_import_testimage
+  ensure_has_localhost_remote "${INCUS_ADDR}"
+
+  incus-user --group nogroup &
+  USER_PID="$!"
+  sleep 3
```

Comment:
Seems fragile - given how slow github runners can sometimes be, it would seem worth checking availability over the socket in a loop, if possible.